src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIResponse.java

```diff
@@ -98,7 +101,24 @@ public int getHttpStatus() {
   }
 
   public boolean isSuccess() {
-    return successCodes.contains(getHttpStatus());
+    boolean isSuccess = true;
+    if (checkRetriable()) {
+      isSuccess = false;
+    } else if (successCodes.contains(getHttpStatus())) {
+      isSuccess = true;
```

Comment:
In case of 404 Http Status, are we considering it as success here ?

Reply:
No. 
The default value for **isSuccess** has been set as `false` bf1facd